src/lib/y2storage/partition_id.rb

```diff
@@ -94,9 +98,10 @@ def formattable?
     # them in the old libstorage.
     # See {.new_from_legacy} and {#to_i_legacy}.
     CURRENT_TO_LEGACY = {
-      BIOS_BOOT.to_i          => 259, # BIOS_BOOT.to_i is 257, that used to mean mac_hidden
+      BIOS_BOOT.to_i          => 263, # BIOS_BOOT.to_i is 257, that used to mean mac_hidden
+      ESP.to_i                => 259, # ESP.to_i is 239, that used to have no special meaning
       WINDOWS_BASIC_DATA.to_i => 0, # WINDOWS_BASIC_DATA.to_i is 258, that used to mean mac_hfs
```

Comment:
I'm actually kind of worried about `WINDOWS_BASIC_DATA`. This is the type used by Windows in GPT. Exporting it as Unknown instead of 258 may be a way to avoid confusions with the old mac_hfs fsid. But it's a recipe for disaster for the future. Let me think it a little bit better. :-)

Reply:
@imobachgs Let me do some experiments with a Windows installation in GPT and the old yast-storage

Reply:
So far, I can't export such partition with AutoYaST. But the old expert partitioner displays it as 0x07. So I guess we need more complex logic in the new AutoYaST to make the `NTFS <=> WINDOWS_BASIC_DATA` conversion based on the type of partition table. fsid really mixed a little bit of everything :-)